test/infrastructure/RepositoryFactory.spec.ts

```diff
@@ -70,6 +72,17 @@ describe('RepositoryFactory', () => {
         expect(repositoryFactory.createFinalizationRepository()).to.be.not.null;
     });
 
+    it('Raise error without unhandled-rejections', async () => {
+        const url = 'https://www.google.com';
```

Comment:
Any other way to achieve this without relying on a remote call? I think we can use `sinon` to stub the `getNodeHealth`method to throw an error.

Reply:
TS SDK uses ts-mockito. I've mocked the low-level generated NodeRoutesApi reflecting how it rejects responses with the current mock lib. The (nodeRepository as any).nodeRoutesApi doesn't look nice, but well.